SPEC/ovirt-openvswitch.spec

```diff
@@ -0,0 +1,153 @@
+%define ovs_version 2.11
+%define ovs_version_to_replace 2.11.1
+
+%global py_openvswitch python3-openvswitch
+
+Name:           ovirt-openvswitch
+Version:        %{ovs_version}
+Release:        0.2020060501%{?dist}
+Summary:        Wrapper RPM for upgrading OVS to newer versions
+
+Group:          System Environment/Daemons
+License:        Public Domain
+URL:            http://www.openvswitch.org
+BuildArch:      noarch
+
+Requires:       openvswitch%{ovs_version}
+Provides:       openvswitch = %{ovs_version}
+Obsoletes:      openvswitch <= %{ovs_version_to_replace}
+Obsoletes:      openvswitch-dpdk <= %{ovs_version_to_replace}
+
+%description
+Wrapper rpm for the openvswitch package
+
+%package -n     ovirt-python-openvswitch
+Summary:        Wrapper for python-openvswitch rpm
+License:        Public Domain
+Requires:       %{py_openvswitch}%{ovs_version}
+Provides:       %{py_openvswitch} = %{ovs_version}
+Obsoletes:      %{py_openvswitch} <= %{ovs_version_to_replace}
+
+%description -n ovirt-python-openvswitch
+Wrapper rpm for the base python-openvswitch package
+
+%package        devel
+Summary:        Wrapper for openvswitch-devel rpm
+License:        Public Domain
+Requires:       openvswitch%{ovs_version}-devel
+Provides:       openvswitch-devel = %{ovs_version}
+Obsoletes:      openvswitch-devel <= %{ovs_version_to_replace}
+
+%description devel
+Wrapper rpm for the base openvswitch-devel package
+
+%package        ovn
+Summary:        Wrapper for ovn rpm
+License:        Public Domain
+Requires:       ovn%{ovs_version}
+Provides:       openvswitch-ovn = %{ovs_version}
+Provides:       ovn = %{ovs_version}
+Obsoletes:      ovn <= %{ovs_version_to_replace}
+
+%description ovn
+Wrapper rpm for the base openvswitch-ovn-central package
+
+%package        ovn-central
+Summary:        Wrapper for openvswitch-ovn-central rpm
+License:        Public Domain
+Requires:       ovn%{ovs_version}-central
+Requires:       ovirt-openvswitch-ovn
+Provides:       openvswitch-ovn-central = %{ovs_version}
+Obsoletes:      openvswitch-ovn-central <= %{ovs_version_to_replace}
+Obsoletes:      ovn-central <= %{ovs_version_to_replace}
+
+%description ovn-central
+Wrapper rpm for the base openvswitch-ovn-central package
+
+%package        ovn-host
+Summary:        Wrapper for openvswitch-ovn-host rpm
+License:        Public Domain
+Requires:       ovn%{ovs_version}-host
```

Comment:
aren't these the d/s packages ? I'm very confused. Where do they come from if not?

Reply:
I want to have them upstream.
You can try via

```
dnf install -y createrepo wget

for task in 1333313 1323726 1322380 1322370 ; do
  curl -k "https://cbs.centos.org/koji/taskinfo?taskID=$task" | grep "https.*el8.*rpm" | sed -e 's/.*a href="\([^"]*\).*/\1/' |  wget -nc -i -
 done

createrepo $PWD
echo -e "[local]\nname = local\nbaseurl = file://$PWD\nenabled = 1\ngpgcheck = 0\n" > /etc/yum.repos.d/local.repo
```